NodeJS/src/RestClient.ts

```diff
@@ -388,6 +394,59 @@ export class RestClient {
       return response.data
     })
   }
+
+  public async getAddressBook() {
+    const subAccountAppId = await this.getSubAccountAppId()
+    const url = `${this.baseUrl}/${ADDRESS_BOOK_PATH}/${subAccountAppId}`
+    const options = this.prepareGetOptions(url)
+    return axios(options).then((response: { data: any }) => {
+      return response.data
+    })
+  }
+
+  public async searchContacts(request?: AddressBookSearchRequest) {
+    const subAccountAppId = await this.getSubAccountAppId()
+    const url = `${this.baseUrl}/${ADDRESS_BOOK_PATH}/search/${subAccountAppId}`
+    let queryParams = ''
```

Comment:
recommend:
```suggestion
    const queryParams = request !== undefined ? buildQueryParams(request) : ''
```

where `buildQueryParams` is a private function of type:
`buildQueryParams(request: AddressBookSearchRequest): string`

Because small and clear functions are sexy as hell

Reply:
👍

Reply:
fixed